plutarch-test/common/Plutarch/Test/Golden.hs

```diff
@@ -201,6 +229,11 @@ data GoldenTest
     Bench
   deriving stock (Eq, Show, Ord, Enum, Bounded)
 
+goldenTestsFromConf :: GoldenConf -> [GoldenTest]
+goldenTestsFromConf (GoldenConf {trackPreEval, trackPostEval, trackBench}) =
+  let goldenKeys = [(trackPreEval, UPLCPreEval), (trackPostEval, UPLCPostEval), (trackBench, Bench)]
+   in [t | (True, t) <- goldenKeys]
```

Comment:
Is this more readable than `do { (True, t) <- goldenKeys ; pure t }` or `goldenKeys >>= \case { (True, t) -> [t] ; _ -> [] }`?

Reply:
The list comprehension looks more natural to me but I don't really care either way. Let me know if you want this changed to bind/do.